### ISSUE TITLE
Club two pretrained Resnets 

### ISSUE THREAD

**Issue by varghesealex90** [pytorch/vision#278]
Hi;

I am using[http://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html](url) as a reference for pre-training a resnet on my data. I would like to know if the following modifications could be made:

1) As per Resnet18 implementation, there are no dropouts in the network. Is it possible to add dropouts in the network and then finetune the network.

2) I have images from two different sources (Camera1 ) and (Camera2). I would like to feed the images from both cameras to 2 different Resnets and club the penultimate layers of both the networks and then do classification. Is such a thing possible.

Thanks in advance for your help in this regard

Varghese

**Comment by alykhantejani** [comment]
Hi @varghesealex90,

In general this style of question is better suited for the [PyTorch Forums](https://discuss.pytorch.org/). In any case here are my thoughts on your questions:

1) You can follow the advice on [this forum post](https://discuss.pytorch.org/t/how-to-finetune-inception-v3/7886) about adding dropout to the inception net and apply it to resnet

2) by "club" I'm assuming you mean concatenate the output features. If you are editing the resnet forward pass anyway you can just return the feature layer that you want and then after passing each image through a separate instance of resnet you can join the features with the [`torch.cat` function](http://pytorch.org/docs/master/torch.html?highlight=cat#torch.cat)
